gen3/lib/g3k_manifest.sh

```diff
@@ -122,6 +122,15 @@ g3k_environment() {
 g3k_hostname > /dev/null 2>&1 || true
 g3k_environment > /dev/null 2>&1 || true
 
+#
+# Lookup and cache slack_webhook 
+#
+g3k_slack_webhook() {
+  if [[ -z "$GEN3_SLACK_WEBHOOK" ]]; then
+    GEN3_CACHE_SLACK_WEBHOOK="$(g3kubectl get configmaps global -ojsonpath='{ .data.slack_webhook }')" || return 1
+  fi
+  echo "$GEN3_CACHE_SLACK_WEBHOOK"
```

Comment:
```suggestion
  echo "GEN3_CACHE_SLACK_WEBHOOK: $GEN3_CACHE_SLACK_WEBHOOK"
```

Reply:
Thats what ends up getting returned from the function, bash is weird, so we want it to just return the url for the terraform config.